test/service/test_manifest.py

```diff
@@ -433,6 +433,17 @@ def test_manifest_zarr(self):
                     # 2 pairs of files out of the 13 have matching checksums
                     self.assertEqual({f['sha256'] for f in expected}, {row['file_sha256'] for row in rows})
                     self.assertEqual({f['crc32c'] for f in expected}, {row['file_crc32c'] for row in rows})
+                    # Manifest for `curl` format  should also contain related files
+                    response = self._get_manifest(ManifestFormat.curl, filters)
+                    self.assertEqual(200, response.status_code)
+                    lines = response.content.decode().splitlines()
+                    output_paths = set()
+                    for line in lines:
+                        if line.startswith('output'):
```

Comment:
I thought I saw in a previous review comment from Hannes that `'output'` be changed to `'output='`?